compute_short_tls:              true    # bool

# Params
max_trace_length:               4       # int   [num]
